contributing/development/compiling/introduction_to_the_buildsystem.rst

```diff
@@ -365,6 +365,23 @@ aforementioned ``-j`` option for all future builds:
 
      $env:SCONSFLAGS="-j4"
 
+SCU (single compilation unit) Build
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Regular builds tend to be bottlenecked by including large numbers of headers in each compilation translation unit. Primarily to speed up development (rather than production builds), Godot offers a "single compilation unit" build (aka "Unity / Jumbo Build").
```

Comment:
```suggestion
Regular builds tend to be bottlenecked by including large numbers of headers in each compilation translation unit. Primarily to speed up development (rather than for production builds), Godot offers a "single compilation unit" build (aka "Unity / Jumbo Build").
```